tests/integration/launch_container.sh

```diff
@@ -26,7 +26,11 @@ if [[ "$platform" == *"-gpu"* ]]; then # if the platform has cuda capabilities
 elif [[ "$platform" == *"lmi"* || "$platform" == *"trtllm"*  || "$platform" == *"tensorrt-llm"* ]]; then # Runs multi-gpu
   runtime="nvidia"
   is_llm=true
-  shm="12gb"
+  if [[ "$platform" == *"trtllm"* || "$platform" == *"tensorrt-llm"*]]; then
+    shm="20gb"
```

Comment:
we should rather check if it is p4d than doing 20gb on all G5 as well. Could we introduce something like p4d or something to make it larger?

Reply:
Added a function to check for that